packages/sit-onyx/src/components/OnyxSliderControl/types.ts

```diff
@@ -0,0 +1,60 @@
+import type { DensityProp } from "../../composables/density.js";
+
+export const SLIDER_CONTROLS = ["icon", "value", "input"] as const;
+export type SliderControl = (typeof SLIDER_CONTROLS)[number];
+
+export type OnyxSliderControlSharedProps = {
+  /**
+   * The type of control to render.
+   *
+   * - `value`: Renders a read-only display of the provided slider value.
+   * - `icon`: Renders an icon button for increasing or decreasing the slider value.
+   * - `input`: Renders an `<OnyxStepper />` component for direct value entry.
+   */
+  control: SliderControl;
+  /**
+   * Slider control model value
+   */
+  modelValue?: number;
+  /**
+   * The read-only value to display inside the control. In the slider context, this is min and max values.
+   */
+  value?: number;
```

Comment:
For the control="value" we can also pass the modelValue property so we can get rid of this additional prop :)